### ISSUE TITLE
feature: equalizer

### ISSUE THREAD

**Issue by jessicajonesunitedkingdom** [emoon/HippoPlayer#101]
an equalizer like winamp

**Comment by emoon** [comment]
Yeah. This is something that needs to be added. Also need to be possible to configure on a per format basis and not just only a global one.

**Comment by jessicajonesunitedkingdom** [comment]
This just awesome!
This is the first open source project I found where the main developer is friendly and REALLY wants to make things and likes suggestions.

I once tried audacious (because it can play some oldskool formats) and asked for some features but they don't even reply ...

**Comment by emoon** [comment]
Thanks! I always appreciate ideas for my projects. The thing is with many of my projects is that I make them because it's something I want myself, but if other finds them useful that is great!

I'm just going to say that many of these things will take time to implement as I have other projects I do as well (and it's all on the spare time also)
